Source/WebCore/platform/graphics/BitmapImageSource.h

```diff
@@ -82,6 +82,7 @@ class BitmapImageSource : public ImageSource {
     DecodingStatus frameDecodingStatusAtIndex(unsigned index) const final;
 
     // Testing support
+    URL sourceURL() const;
```

Comment:
This says "testing support" but we always call `sourceURL` for async decoding (but only to log it)

Reply:
This new function is removed.